testhelper/tests.go

```diff
@@ -25,11 +24,27 @@ import (
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/common"
 )
 
-func skipUpgradeTest(branch string) bool {
+func (options *TestOptions) skipUpgradeTest(source_repo string, branch string) bool {
+
+	// Set upstream to the source repo
+	remote_out, remote_err := exec.Command("/bin/sh", "-c", "git remote add upstream "+source_repo).Output()
+	if remote_err != nil {
+		logger.Log(options.Testing, "Add remote output:\n", remote_out)
+		logger.Log(options.Testing, "Error adding upstream remote:\n", remote_err)
+		return false
+	}
+	// Fetch the source repo
+	fetch_out, fetch_err := exec.Command("/bin/sh", "-c", "git fetch upstream -f").Output()
+	if fetch_err != nil {
```

Comment:
Did you want to do anything here if the fetch fails?

Reply:
I don't think so, I was just going to run the upgrade test if it cannot figure it out and log a message. I feel this is better than killing the test... But its open for discussion